### PR TITLE
Brothers ENM

### PR DESCRIPTION
**Pull request by wrenffxi**
* add mob HP, NM flag, resistances, apocalyptic ray, draw-in
* add chance to drop nothing to the loot groups

### CODE REVIEW

File: scripts/zones/Bearclaw_Pinnacle/mobs/Eldertaur.lua [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+-----------------------------------
+
+function onMobInitialize(mob)
+    mob:setMobMod(dsp.mobMod.DRAW_IN, 1)
+    mob:setMod(dsp.mod.DMGMAGIC, -10)
+    mob:setMod(dsp.mod.SLEEPRES, 75)
+end
+
+function onMobSpawn(mob)
+    mob:setMobMod(dsp.mobMod.SUPERLINK, mob:getShortID())
```
Comment (TeoTwawki) [inline scripts/zones/Bearclaw_Pinnacle/mobs/Eldertaur.lua]: @teschnei didn't we have this being handled automatically for bcnm mobs without having to set this?

Comment (TeoTwawki) [reply scripts/zones/Bearclaw_Pinnacle/mobs/Eldertaur.lua]: We should fix that if it broke

Comment (zynjec) [reply scripts/zones/Bearclaw_Pinnacle/mobs/Eldertaur.lua]: Probably because of https://github.com/DarkstarProject/darkstar/issues/6119

Comment (TeoTwawki) [reply scripts/zones/Bearclaw_Pinnacle/mobs/Eldertaur.lua]: yeah ok, we needa fix that then after we'll nuke all these manual superlinks. waiting on appveyor atm to merge this.

**Pull request merged**